### PR TITLE
Fix lint errors an broken tests

### PR DESCRIPTION
**Pull request by Stebalien**
Note: this blindly changes some test constants. Those changes need to be verified by someone who understands them.

Most of these changes are small fixes to get `go vet ./...` passing.

### CODE REVIEW

File: extern/sector-storage/go.mod [hunk 1]
```diff
@@ -23,10 +23,11 @@ require (
...
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
-	golang.org/x/sys v0.0.0-20200317113312-5766fd39f98d // indirect
-	golang.org/x/tools v0.0.0-20200318150045-ba25ddc85566 // indirect
 	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
 	honnef.co/go/tools v0.0.1-2020.1.3 // indirect
 )
+
+replace github.com/filecoin-project/storage-fsm => ../storage-fsm
+
+replace github.com/filecoin-project/filecoin-ffi => ../filecoin-ffi
```
Comment (Stebalien) [inline extern/sector-storage/go.mod:33]: Change.

File: extern/storage-fsm/go.mod [hunk 2]
```diff
@@ -22,3 +22,7 @@ require (
 )
 
 replace github.com/golangci/golangci-lint => github.com/golangci/golangci-lint v1.18.0
+
+replace github.com/filecoin-project/sector-storage => ../sector-storage
+
+replace github.com/filecoin-project/filecoin-ffi => ../filecoin-ffi
```
Comment (Stebalien) [inline extern/storage-fsm/go.mod:28]: Change.

File: extern/storage-fsm/precommit_policy_test.go [hunk 3]
```diff
@@ -29,7 +37,7 @@ func TestBasicPolicyEmptySector(t *testing.T) {
 	exp, err := policy.Expiration(context.Background())
 	require.NoError(t, err)
 
-	assert.Equal(t, 3455, int(exp))
+	assert.Equal(t, 2879, int(exp))
```
Comment (Stebalien) [inline extern/storage-fsm/precommit_policy_test.go:40]: All of these constants (this and below) needed to be changed to get the tests to pass. Is this correct?

File: extern/sector-storage/go.mod (hunk 1 above)
Comment (vyzo) [inline extern/sector-storage/go.mod:33]: uhm, is this intended? why is the go.mod changing?

Comment (Stebalien) [reply extern/sector-storage/go.mod:33]: Otherwise, it's impossible to build/vet/test these modules.

Comment (whyrusleeping) [reply extern/sector-storage/go.mod:33]: this

**Comment by whyrusleeping** [comment]
@Stebalien delete your go mod changes

File: extern/storage-fsm/precommit_policy_test.go (hunk 3 above)
Comment (magik6k) [reply extern/storage-fsm/precommit_policy_test.go:40]: I'm going to guess that we used very old actors in storage-fsm, this is probably fine

**Pull request merged**